src/Symfony/Component/Validator/Tests/Validator/AbstractTest.php

```diff
@@ -528,7 +528,7 @@ public function testAddCustomizedViolation()
         $this->assertSame($entity, $violations[0]->getRoot());
         $this->assertSame('Invalid value', $violations[0]->getInvalidValue());
         $this->assertSame(2, $violations[0]->getPlural());
-        $this->assertSame(42, $violations[0]->getCode());
+        $this->assertSame('42', $violations[0]->getCode());
```

Comment:
The argument is documented as `string` in `ConstraintViolationBuilderInterface::setCode()`.  When adding type-hints in 5.0, we will need to compare the returned code with the string `'42'`. Making the change in the test here makes it consistent across all branches.

Reply:
ref. https://github.com/symfony/symfony/pull/32066#discussion_r294106130